infra/terraform/test-org/org/locals.tf

```
@@ -743,6 +743,15 @@ locals {
       groups          = ["dee-data-ai", local.jss_common_group]
       enable_periodic = true
     },
+    {
+      name            = "terraform-genai-knowledge-base"
+      short_name      = "genai-doc-knowledge-base"
```

This just needs to match the value in `infra/terraform/test-org/ci-triggers/locals.tf`
```suggestion
      short_name      = "genai-knowledge-base"
```

---

@NimJay